app/views/layouts/base.html.erb

```diff
@@ -3,9 +3,11 @@
 
 <head>
   <meta charset="utf-8" />
-  <meta content="<%= APP_NAME %>" name="description" />
-  <meta content="width=device-width, initial-scale=1.0" name="viewport" />
-  <meta content="telephone=no" name="format-detection" />
+  <meta name="description" content="<%= APP_NAME %>" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="format-detection" content="telephone=no" />
+  <meta name="theme-color" content="#ffffff" />
+  <meta name="og:site_name" content="<%= APP_NAME %>" />
```

Comment:
Should we add a regression spec?

Reply:
I thought about it. I even thought about pulling in a library like [ogp](https://github.com/jcouture/ogp) to assert against the parsed metadata. Without a library, it felt a little like testing the implementation detail, but I'm not opposed to it just to say we have the coverage.

Reply:
I'm fine with adding a gem like but I think [Mitchell tends to prefer fewer deps](https://github.com/18F/identity-idp/pull/10834#discussion_r1647637101). In my mind the goal of adding a simple regression is that if we ever reorganize or refactor some of our layouts, we'd want to have a spec as a guardrail

Reply:
I do lean away from adding dependencies where our use-case is more limited relative to the gem. I'd probably rather avoid it, but not strongly so. This one is relatively short and makes heavy use of OpenStruct, which I'd probably prefer to avoid if we were using it outside of tests: https://github.com/jcouture/ogp/blob/1e6f601/lib/ogp/open_graph.rb#L55-L103

Reply:
Added a simple test in f820552.